pkg/controllers/binding/binding_controller.go

```diff
@@ -69,7 +69,7 @@ func (c *ResourceBindingController) Reconcile(ctx context.Context, req controlle
 		return c.removeFinalizer(binding)
 	}
 
-	isReady := helper.IsBindingReady(&binding.Status)
+	_, isReady := binding.Annotations[util.PolicyPlacementAnnotation]
```

Comment:
It looks weird because why has the annotation is `ready`?

Reply:
Precisely it does NOT mean schedule successed, but has been **scheduled once**

We have to aggregate status no matter if the schedule successed or not, don't we?

Reply:
Please look at the latest code. This check has been postponed to `syncBinding`.